apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -75,7 +75,7 @@ export default function manageStudents(state=initialState, action) {
         ...state.editingData,
         [action.studentId]: {
           ...state.studentData[action.studentId],
-          id: action.studentData,
+          id: action.studentId,
```

Comment:
this fix looks good, though hard to understand how it was ever working beforehand 🤔

Reply:
I think an accidental commit made it into my last PR. Going to do a follow up testing PR because it's concerning.